### ISSUE TITLE
proxy support

### ISSUE THREAD

**Issue by steom** [gravitl/netmaker#1446]
please consider to add socks5 or http proxy support for the client

**Comment by Nike682631** [comment]
is this issue available to work on?

**Comment by afeiszli** [comment]
certainly! though this would be a very big change I believe, honestly not sure where to get started.
